src/Algebra/Definitions/RawMonoid.agda

```diff
@@ -65,3 +66,18 @@ suc n ×′ x = n ×′ x + x
 
 sum : ∀ {n} → Vector Carrier n → Carrier
 sum = Vector.foldr _+_ 0#
+
+------------------------------------------------------------------------
+-- 'Conjunction' with a Boolean: action of the Boolean (true,∧)-rawMonoid
+------------------------------------------------------------------------
+
+infixr 8 _∧_
+
+_∧_ : Bool → Carrier → Carrier
+b ∧ x = if b then x else 0#
```

Comment:
Strictly speaking, this could be defined for any `Pointed` structure with an apartness relation, as an action on a `Pointed` structure.

How is the above 'conjuction'?

Reply:
OK, I think that's fair... I was being pretty *ad hoc* in wanting to add this by analogy with the `Nat` action already defined there, as then these two instances of a phenomenon (eventually: wreath product) are the relevant ones for the `Algebra.Solver.*CommutativeMonoid` refactorings #2407 #2457 ... and I'm still unhappy about the general picture for `Pointed` things in the library, so don't (yet) want to reopen that can of worms...

As for it being 'conjunction', well it clearly generalises it: for `Carrier` being that of the `Monoid` structure on `Bool` given by `_∨_` with unit `false`, 
```agda
b ∧ x = if b then x else false
```
is a definition of 'conjunction'... yes!?

Reply:
Yes, that is the operational definition of 'conjunction' on `Bool`. 

I guess that I was thinking along the lines of "if b is the least element then x else some-distinguished-point" as the general meaning. It feels more action-y. 'conjunction' feels like it's a coincidence when `Carrier` is `Bool`.